src/platform/forms-system/src/js/review/submit-states/ClientError.jsx

```diff
@@ -7,20 +7,13 @@ import PropTypes from 'prop-types';
 import { Column, Row } from 'platform/forms/components/common/grid';
 import ErrorMessage from 'platform/forms/components/common/alerts/ErrorMessage';
 import PreSubmitSection from 'platform/forms/components/review/PreSubmitSection';
+import scrollTo from 'platform/utilities/ui/scrollTo';
 
 export default function ClientError(props) {
   const { buttonText, formConfig, onBack, onSubmit, testId } = props;
   const Element = Scroll.Element;
-  const scroller = Scroll.scroller;
   const scrollToError = () => {
-    scroller.scrollTo(
-      'errorScrollElement',
-      window.VetsGov?.scroll || {
-        duration: 500,
-        delay: 0,
-        smooth: true,
-      },
-    );
+    scrollTo('errorScrollElement', window.VetsGov?.scroll || '');
```

Comment:
What is the impact of the object literal vs the empty string in the second argument?

Reply:
The centralized function that's being imported from the `utilities/ui` directory has those same args as the default args (which are coming from `getScrollOptions`. The main objective we're trying to get away from is the applications each sending in hard-coded args when they're often all the same (duration 500, delay 0, smooth: true). So in this case, the blank string will cause the helper function to fall back on what was the same object literal, it's just avoiding a duplicate declaration of that object. Eventually, the goal is to implement smooth scrolling to be conditional based on the user's OS preference for reduced scrolling:

department-of-veterans-affairs/va.gov-team#6844 centralized scrolling ticket

https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/312 smooth scrolling ticket

Reply:
How will the empty string cause the helper function to fall back on the object literal? The way I'm reading things, we just end up with:

```js
scroller.scrollTo(elem, '')
```

https://github.com/department-of-veterans-affairs/vets-website/blob/eb93f6f939f5866e8252fac497f95925a34ba798/src/platform/utilities/ui/scrollTo.js#L6-L8

The empty string will override the default parameter since a value is being passed and it isn't `undefined`.

Reply:
I tested this logging out both the element and options from the helper function and when passed the empty string, it logged out the default object literal. I'm happy to go back and test again though with `undefined` or `null` passed instead and see if the result is the same if that's deemed to be a more stable option.

Reply:
If it behaves the same without `|| ''`, I would be fine with that. The optional chaining should ensure that `undefined` gets passed, and I think that is more clear.

Reply:
Changed this to undefined and tested it seems to work the same as the blank string, so I've updated it to reflect that. Should be all resolved here!